src/parse/asp/main/main.go

```diff
@@ -31,6 +32,7 @@ var opts = struct {
 	NumThreads   int           `short:"n" long:"num_threads" default:"10" description:"Number of concurrent parse threads to run"`
 	ParseOnly    bool          `short:"p" long:"parse_only" description:"Only parse input files, do not interpret them."`
 	DumpAst      bool          `short:"d" long:"dump_ast" description:"Prints AST to stdout. Implies --parse_only."`
+	Check        bool          `short:"c" long:"check" description:"Runs some static checks on the parsed files. Implies --parse_only."`
```

Comment:
Can we just set this as part of `plz fmt`?

Reply:
Yeah that might be a good place for it to live. It's a _little_ weird since it's not actually a format thing though...